CRM/Event/Form/Participant.php

```diff
@@ -1643,11 +1631,6 @@ public function submit($params) {
       }
     }
 
-    // set the participant id if it is not set
```

Comment:
@eileenmcnaughton i think this is important for the postProcess hook right? i.e. in postProcess you expect that $this->_id is set i.e. if we are adding a Participant Registration here then it would be NULL in the preProcess but then should be set here now that we have created the Participant registration

Reply:
or are you saying it is duplicate of https://github.com/civicrm/civicrm-core/pull/16324/files#diff-63338fa600c17c14a171ac3033cccdedR1540 ?